README.md

```diff
@@ -1,32 +1,50 @@
-# API Elements JavaScript
+# API Elements
 
-This library provides an interface to [API Elements](http://apielements.org) in JavaScript.
+## Using Lerna
 
-## Install
+Install dependencies of all packages:
 
 ```shell
-$ npm install api-elements
+$ npm install --global yarn
+$ yarn
 ```
 
-## Usage
+> **NOTE**: Please do commit the yarn.lock to the github repo
 
-### Node
+To list all packages:
 
-```javascript
-const apiElements = require('api-elements');
-const namespace = new apiElements.Namespace();
+```shell
+$ npx lerna ls -a -l
+```
+
+To add a new dependency to a package:
+
+```shell
+$ npx lerna add --scope='package_name' dep@version
+```
+
+To run tests for a single package:
 
+```shell
+$ npx lerna exec --scope='package_name' -- npm run test
+```
 
-// Parsing a JSON Representation of API Elements tree
-const parseResult = namespace.serialiser.deserialise({
-  element: 'parseResult',
-  content: []
-});
+## Documentation
 
-console.log(parseResult);
+The documentation is built using Sphinx, a Python tool. Assuming you have
+Python 3 installed, the following steps can be used to build the site.
 
+```shell
+$ python3 -m venv venv
+$ source venv/bin/activate
+$ cd docs/
+$ pip install -r requirements.txt
+```
 
-// Creating API Elements directly
-const parseResult = new namespace.elements.ParseResult();
-console.log(parseResult);
+### Running the Development Server
+
+You can run a local development server to preview changes using the following:
+
+```shell
+$ make watch
```

Comment:
Wouldn't it be nicer to have npm scripts for this? Also, I haven't checked the `requirements.txt`, but if you use the same dependencies as in https://github.com/apiaryio/dredd/pull/1119, they're all approved and then you can basically copy-paste the Dredd setup, if you like it.

https://github.com/apiaryio/dredd/blob/c2dadfbd9921725f817f23783a3ee15ca6753546/package.json#L18:L19

Up to you if this is something for a follow-up PR, I don't want to block on this one.

Reply:
It would, https://github.com/apiaryio/api-elements.js/issues/17 now mentions integrating the docs with NPM.